### PR TITLE
fix design, write test for it

### PR DESCRIPTION
**Pull request by dendi239**
link #6

### CODE REVIEW

File: pkg/poll/poll.go [hunk 1]
```diff
@@ -76,11 +76,19 @@ var (
 			},
 		},
 	}
+	vika = UserOptions{
+		"@viskonsin",
```
Comment (vikotia) [inline pkg/poll/poll.go]: really nice design, i like it

File: pkg/poll/poll.go [hunk 2]
```diff
@@ -76,11 +76,19 @@ var (
...
+	}
 	// Sample poll to test interface
 	Sample = Poll{
 		"DAF Pub",
 		[]int{1, 2, 3, 4, 5, 6},
-		[]UserOptions{daniel, denys, belkka},
+		[]UserOptions{daniel, denys, belkka, vika},
 		map[int]string{
 			1: "tue 20:00",
 			2: "tue 21:00",
```
Comment (vikotia) [inline pkg/poll/poll.go]: put it in the test file please

**Review by vikotia — APPROVED** [review]
lgtm

File: pkg/poll/poll.go (hunk 1 above)
Comment (dendi239) [reply pkg/poll/poll.go]: Me too!

File: pkg/poll/poll.go (hunk 2 above)
Comment (dendi239) [reply pkg/poll/poll.go]: done

**Pull request merged**